### ISSUE TITLE
2 test failures in Epoll in ppc64le docker container

### ISSUE THREAD

**Issue by seth-priya** [netty/netty#11557]
### Expected behavior
All unit tests should pass

### Actual behavior
2 test failure in Netty/Transport/Native/Epoll inside UBI 8.4 container on ppc64le
[INFO]
[ERROR] Failures:
[ERROR]   EpollDatagramUnicastTest.testSendAndReceiveSegmentedDatagramPacketComposite:101->AbstractComboTestsuiteTest.run:52->testSendAndReceiveSegmentedDatagramPacketComposite:110->testSegmentedDatagramPacket:158 expected: <true> but was: <false>
[ERROR]   EpollDatagramUnicastTest.testSendAndReceiveSegmentedDatagramPacket:87->AbstractComboTestsuiteTest.run:52->testSendAndReceiveSegmentedDatagramPacket:96->testSegmentedDatagramPacket:158 expected: <true> but was: <false>
[INFO]
[ERROR] Tests run: 753, Failures: 2, Errors: 0, Skipped: 5

### Steps to reproduce
1. Build netty-tcnative from source (mvn install)
2. Build netty from source (mvn install)

### Minimal yet complete reproducer code (or URL to code)
NA

### Netty version
master

### JVM version (e.g. `java -version`)
openjdk version "1.8.0_302"
OpenJDK Runtime Environment (build 1.8.0_302-b08)
OpenJDK 64-Bit Server VM (build 25.302-b08, mixed mode)

### OS version (e.g. `uname -a`)
Linux 9d7f20f28a85 4.18.0-240.22.1.el8_3.ppc64le #1 SMP Thu Mar 25 14:33:26 EDT 2021 ppc64le ppc64le ppc64le GNU/Linux

**Comment by chrisvest** [comment]
Are you consistently getting this test failure on repeated runs? Some of our tests are flaky and fail on occasion.

**Comment by seth-priya** [comment]
@chrisvest yes these are consistent. I am seeing flakiness in some of the others , especially when running in container vs VM host environment but these 2 seem to stick. I also see these with previous stable versions like netty-4.1.63.Final where everything else is passing consistently, these almost seem like networking issue but I am not sure what would be the best place to debug / things to check.

**Comment by chrisvest** [comment]
You're saying things broke in 4.1.64? There are some epoll changes, but not many: https://github.com/netty/netty/compare/netty-4.1.63.Final...netty-4.1.64.Final

**Comment by seth-priya** [comment]
Sorry, no I meant these epoll tests are failing consistent even in netty-4.1.63.Final  (that's where I started)
